### PR TITLE
refactor(client)!: Remove request wrappers - LeaseClient::*

### PR DESCRIPTION
**Pull request by lxl66566**
Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)

* what changes does this pull request make?

* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)

### CODE REVIEW

File: crates/xline-client/src/clients/lease.rs [hunk 1]
```diff
@@ -88,19 +88,22 @@ impl LeaseClient {
...
     /// ```
     #[inline]
-    pub async fn grant(&self, mut request: LeaseGrantRequest) -> Result<LeaseGrantResponse> {
-        if request.inner.id == 0 {
-            request.inner.id = self.id_gen.next();
+    pub async fn grant(&self, ttl: i64, id: Option<i64>) -> Result<LeaseGrantResponse> {
+        let mut id = id.unwrap_or_default();
+        if id == 0 {
+            id = self.id_gen.next();
         }
```
Comment (LucienY01) [inline crates/xline-client/src/clients/lease.rs:102]: We can use `unwrap_or_else` here.

File: crates/xlinectl/src/command/lease/grant.rs [hunk 2]
```diff
@@ -11,15 +11,15 @@ pub(super) fn command() -> Command {
...
-pub(super) fn build_request(matches: &ArgMatches) -> LeaseGrantRequest {
+pub(super) fn build_request(matches: &ArgMatches) -> i64 {
     let ttl = matches.get_one::<i64>("ttl").expect("required");
-    LeaseGrantRequest::new(*ttl)
+    *ttl
 }
 
 /// Execute the command
 pub(super) async fn execute(client: &mut Client, matches: &ArgMatches) -> Result<()> {
     let request = build_request(matches);
```
Comment (LucienY01) [inline crates/xlinectl/src/command/lease/grant.rs:21]: Please rename `request` `ttl`.

File: crates/xline-client/src/clients/lease.rs (hunk 1 above)
Comment (lxl66566) [reply crates/xline-client/src/clients/lease.rs:102]: if id = `Some(0)`, it needs to be generated as well. `unwrap_or_else` may not deal with that.

Comment (LucienY01) [reply crates/xline-client/src/clients/lease.rs:102]: You are right. I didn't take this case into account.

**Review by LucienY01 — APPROVED** [review]

**Pull request merged**